etc/env/configs/via.toml

```diff
@@ -0,0 +1,63 @@
+__imports__ = [ "base", "l2-inits/dev.init.env" ]
+
+[via_btc_watch]
+confirmations_for_btc_msg=3
+btc_node_poll_interval=300
+rpc_url = "http://0.0.0.0:18443"
+rpc_user = "rpcuser"
+rpc_password = "rpcpassword"
+network = "regtest"
+bootstrap_txids = []
+actor_role = "Sequencer"
+
+[via_btc_sender]
+poll_interval = 1
```

Comment:
This is the time required to restart an execution cycle in btc_sender tasks. The value is in milliseconds, let's set a bigger value, I think for testing 30sec (30000) is good.